### PR TITLE
vol2vox / vox2vol converters 

### PR DESCRIPTION
**Pull request by dcoeurjo**
# PR Description

VOX file converters (https://ephtracy.github.io)

# Checklist

- [x] Doxygen documentation of the code completed (classes, methods, types, members...).
- [x] Main tool doxygen documentation (following existing documentation of [DGtalTools documentation](http://dgtal.org/doc/tools/nightly/).
- [x] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools/blob/master/CONTRIBUTING.md)
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools/blob/master/ChangeLog.md) added.
- [x] Update the readme with potentially a screenshot of the tools if it applies. 
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).

### CODE REVIEW

File: converters/vox2vol.cpp [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+  
+  DGtal::uint32_t XYZI= toInt('X','Y','Z','I');
+  read_word(myfile,version);
+  while ( version != XYZI)
+    read_word(myfile,version);
+  
+  //trash two ints
+  read_word(myfile,version);
+  read_word(myfile,version);
+  uint32_t cpt;
```
Comment (kerautret) [inline converters/vox2vol.cpp]: DGtal:: is missing before uint32_t it will fix the appveyor compilation ;)

**Comment by kerautret** [comment]
Nice new tools thanks that could be nice to import or edit indirectly new Dgtal objects.
Just some comments and can be merged (tested ok with last version of magicavoxel)

**Comment by dcoeurjo** [comment]
Yeah it shouldn’t be too hard to have the reader/writer in Dgtal ;)
At this point, the VOX support is rather limited (no colormap, single chunk..)

File: converters/vol2vox.cpp [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+ 
+ @b Allowed @b options @b are:
+ 
+ @code
+ -h [ --help ]                   display this message
+ -i [ --input ] arg              Input vol file.
+ -o [ --output ] arg             Ouput vox file.
+ @endcode
+ 
+ @b Example:
```
[the remark this answers is not in the document]
Comment (dcoeurjo) [reply converters/vol2vox.cpp]: I should have mentioned that: the VOX is limited to 126^3 domains.
It is probably the issue with lobster

[the remark this answers is not in the document]
Comment (kerautret) [reply converters/vol2vox.cpp]: yes I think so

**Comment by kerautret** [comment]
@dcoeurjo I fixed the conflict, if you can add the mention on size limits, I think that we could merge ;)

File: converters/vol2vox.cpp [hunk 3]
```diff
@@ -127,7 +127,14 @@ int main(int argc, char**argv)
   trace.beginBlock("Loading..");
   MyImageC  imageL = VolReader< MyImageC >::importVol ( filename );
   trace.endBlock();
-
+ 
+  if ( (imageL.domain().upperBound() - imageL.domain().lowerBound()).max() > 126 )
+  {
+    trace.error() << "Vol file to large (width > 126)."<<std::endl;
```
Comment (kerautret) [inline converters/vol2vox.cpp]: "too"  ;)

**Comment by dcoeurjo** [comment]
reum.. thx

**Comment by kerautret** [comment]
perfect thanks, merging

**Pull request merged**